src/openapi/streaming/subscription.ts

```diff
@@ -1159,7 +1163,7 @@ class Subscription {
         }
 
         // Serialization of Snapshot is not yet supported.
-        this.onUpdate?.(response.Snapshot, this.UPDATE_TYPE_SNAPSHOT);
+        this.onUpdate?.(response.Snapshot, this.UPDATE_TYPE_SNAPSHOT, this);

```

Comment:
nitpick - why those types are repeated within the class itself even though they are already defined outside as:

const updateTypes = {
    UPDATE_TYPE_SNAPSHOT: 1,
    UPDATE_TYPE_DELTA: 2,
} as const;

Reply:
I don't know but I don't have much of a preference myself. I guess this way it is consistent with how it is checked (consumers use subscription.UPDATE_TYPE_SNAPSHOT). I agree its a bit messy and would be better exposed separately, but  I think its a bigger standalone task.